docs/user-guide/parameter-configuration.md

```diff
@@ -57,39 +57,39 @@ rosservice call /param_write
 ```
 !!! error
-    Parameter writing can only happen if the flight controller is disarmed.  If the param write failed for some reason, you may want to make sure you are disarmed and try again.
+    Parameter writing can only happen if the flight controller is disarmed. If the param write failed for some reason, you may want to make sure your FC is disarmed and try again.
 
 ### Backing Up and Loading Parameters from File
 
-It is good practice to backup your parameter configuration in case you have to re-flash your firmware or you want to share configurations between vehicles.  We can do this via the `param_save_to_file` and `param_load_from_file` services.
+It is good practice to backup your parameter configuration in case you have to re-flash your firmware or you want to share configurations between vehicles. We can do this via the `param_save_to_file` and `param_load_from_file` services.
 
 First, let's back up our current parameter configuration:
 
 ```
 rosservice call /param_save_to_file ~/parameters.yml
 ```
 
-Parameters are saved in YAML format.  You must also specify the absolute file name of where you would like your parameters to be saved.  The current active set of parameters will be saved, regardless of what is saved in non-volatile memory on the flight controller.
+Parameters are saved in YAML format. You must also specify the absolute file name of where you would like your parameters to be saved. The current active set of parameters will be saved, regardless of what is saved in non-volatile memory on the flight controller.
 
 Now, let's say we want to re-load this parameter file
 ```
 rosservice call /param_load_from_file ~/parameters.yml
 ```
-Again, you must specify the absolute file name of the file to be loaded
+Again, you must specify the absolute file name of the file to be loaded.
 
 
 ## Fixed-Wing Parameter Configuration
 
-Because ROSflight ships with default parameters for multirotors, you will probably want to change the following parameters if you want to fly a fixed wing aircraft.
+Because ROSflight ships with default parameters for multirotors, you will probably want to change the following parameters if you want to fly a fixed-wing aircraft.
 
 
-| Parameter | Description | Type | Fixed Wing Value
+| Parameter | Description | Type | Fixed-Wing Value
 |-----------|-------------|------|---------------|
 | MOTOR_PWM_UPDATE | Refresh rate of motor commands to motors and servos (Hz) - See motor documentation | int |  50 |
 | ARM_SPIN_MOTORS | Enforce MOTOR_IDLE_PWM | int |  false |
 | MOTOR_IDLE_THR | min throttle command sent to motors when armed (Set above 0.1 to spin when armed) | float |  0.1 |
 | ARM_CHANNEL | RC switch channel mapped to arming [0 indexed, -1 to disable] | int |  4 |
-| FIXED_WING | switches on passthrough commands for fixedwing operation | int |  true |
+| FIXED_WING | switches on passthrough commands for fixed-wing operation | int |  true |
```

Comment:
The parameter file is auto-generated from `params.cpp`.  These changes will get overwritten if we don't make the changes there.